integration_tests/e2e/reviewPlan/reviewPlan.cy.ts

```diff
@@ -94,7 +94,7 @@ Working in the prison kitchen is suiting Daniel well and is allowing him to focu
 We have agreed and set a new goal, and the next review is 1 year from now.   
 `,
       )
-    // .submitPage()
+      .submitPage()
```

Comment:
This cypress test is named `should complete a prisoner's review, triggering validation on every screen`, and the code above triggers the validation on the fields on the `WhoCompletedReviewPage` page before correcting them and submitting
Could you do the same thing here please? IE, arrive on the Review Notes page and submit without entering any notes. Verify we are back on the ReviewNotes page with 1 field in error and that it is the correct/expected field; enter a value then submit.

We don't need to trigger ever permutation of validation (that is what the unit tests in the validator class are for), so submitting the page without a value is sufficient 👍